src/test/kotlin/org/rust/lang/core/completion/RsCompletionTestBase.kt

```diff
@@ -35,6 +35,38 @@ abstract class RsCompletionTestBase : RsTestBase() {
         }
     }
 
+    protected fun checkSingleCompletionWithMultipleFiles(target: String, @Language("Rust") code: String) {
+        val files = ProjectFile.parseFileCollection(code)
+        for ((path, text) in files) {
+            myFixture.tempDirFixture.createFile(path, replaceCaretMarker(text))
+        }
+
+        openFileInEditor(files[0].path)
+
+        val variants = myFixture.completeBasic()
```

Comment:
From here and down to the end of the method everything looks like a copy of `checkSingleCompletion()`. It seems that this block is a good candidate to be extracted into a separate method to remove duplication.